docs/support-bundle-guide.md

```diff
@@ -0,0 +1,255 @@
+"worker2". In addition, we have set up two external Nodes named "vm1" and "vm2"
+in the "vm-ns" Namespace by following the instruction of the [VM installation guide](external-node.md#install-antrea-agent-on-vm).
+
+A Secret needs to be created in advance with the username and password of the SFTP
+Server. The Secret will be referred as `authSecret` in the following YAML examples.
+
+```bash
+kubectl create secret generic support-bundle-secret  --from-literal=username='your-sftp-username'  --from-literal=password='your-sftp-password'
+```
+
+Then we can apply the following YAML files. The first one is to collect support
+bundle on K8s Nodes "worker1" and "worker2", and "worker2" is labeled with
+"app: nginx". The second one is to collect support bundle on external Nodes
+"vm1" and "vm2" in Namespace "vm-ns", and ExternalNode "vm2" is labeled with
+"app: db".
+
+```bash
+cat << EOF | kubectl apply -f -
+kind: SupportBundleCollection
+metadata:
+  name: support-bundle-for-nodes
+spec:
+  nodes:
+    nodeNames: # Empty list means all the Nodes
+      - worker1
+    matchLabels:
+      app: nginx
+  expirationMinutes: 10 # expirationMinutes is the requested duration of validity of the SupportBundleCollection. A SupportBundleCollection will be marked as Failed if it does not finish before expiration.
+  sinceTime: 2h # Collect the logs in the latest 2 hours. Collect all available logs if the time is not set.
+  fileServer:
+    url: sftp://yourtestdomain.com:22/root/test
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default # antrea-controller must be given the permission to read Secrets in "default" Namespace. 
+EOF
+```
+
+```bash
+cat << EOF | kubectl apply -f -
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: support-bundle-for-vms
+spec:
+  externalNodes:
+    nodeNames: # Empty list means all ExternalNodes in the Namespace
+      - vm1
+    nodeSelector:
+      matchLabels:
+        app: db
+    namespace: vm-ns # namespace is mandatory when collecting files from external Nodes.
+  fileServer:
+    url: yourtestdomain.com:22/root/test # Scheme sftp can be omitted
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default # antrea-controller must be given the permission to read Secrets in "default" Namespace.
+EOF
+```
+
+For more information about the supported fields in a "SupportBundleCollection"
+CR, please refer to the [CRD definition](/build/charts/antrea/crds/supportbundlecollection.yaml)
+
+You can check the status of `SupportBundleCollection` by running
+`kubectl get supportbundlecollections $Name-of-SupportBundleCollection -ojson`.
+The following example shows a successful realization of `SupportBundleCollection`.
+`desiredNodes` shows the number of Nodes/ExternalNodes to collect information
```

Comment:
"the expected number of"

Reading it again, guess we should remove "information".